### PR TITLE
Style format remarklint

### PR DESCRIPTION
**Pull request by krishguptadev**
#### Checklist

- [ ] Added description of the change
- [ ] I've read the [contributing guidelines](https://github.com/web3community/devprotocol.xyz/blob/main/CONTRIBUTING.md)
- [ ] Search previous suggestions before making a new PR, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.

Notes (**optional**): 

<a href="https://gitpod.io/#https://github.com/web3community/devprotocol.xyz/pull/138"></a>

### CODE REVIEW

File: README.md
Comment (KukretiShubham) [inline README.md:6]: Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8405536)
```suggestion
* Source code has support for writing in **Vue**, **React**, **Svelte**, and/or **Preact**
```

File: CONTRIBUTING.md
Comment (KukretiShubham) [inline CONTRIBUTING.md:64]: Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8405536)
```suggestion
* Please create an [issue](https://github.com/web3community/devprotocol.xyz/issues) before creating a pull request.
```

File: CODE_OF_CONDUCT.md
Comment (KukretiShubham) [inline CODE_OF_CONDUCT.md:22]: Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8405536)
```suggestion
* Giving and gracefully accepting constructive feedback
```

File: CONTRIBUTING.md
Comment (KukretiShubham) [inline CONTRIBUTING.md:57]: Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8405536)
```suggestion
* Wrap the body at *72 characters*.
```

File: CONTRIBUTING.md
Comment (KukretiShubham) [inline CONTRIBUTING.md:128]: Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8405536)
```suggestion
* Comment on the issue to get assigned
```

File: README.md
Comment (KukretiShubham) [inline README.md:10]: Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8405536)
```suggestion
* Accessible, Performant, and SEO Optimized
```

File: CONTRIBUTING.md
Comment (KukretiShubham) [inline CONTRIBUTING.md:55]: Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8405536)
```suggestion
* Separate subject from body with a blank line.
```

File: CONTRIBUTING.md
Comment (KukretiShubham) [inline CONTRIBUTING.md:42]: Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8405536)
```suggestion
* Engage the community in working toward the best possible!
```

File: README.md
Comment (KukretiShubham) [inline README.md:32]: Codacy has a fix for the issue: [Expected: 1; Actual: 2](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8405536)
```suggestion
4. Start the app
```

File: CODE_OF_CONDUCT.md
Comment (KukretiShubham) [inline CODE_OF_CONDUCT.md:33]: Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8405536)
```suggestion
* Public or private harassment
```

File: CONTRIBUTING.md
Comment (KukretiShubham) [inline CONTRIBUTING.md:51]: Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8405536)
```suggestion
* Use the present tense. (Example: "Add feature" instead of "Added feature")
```

File: CONTRIBUTING.md
Comment (KukretiShubham) [inline CONTRIBUTING.md:58]: Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8405536)
```suggestion
* Use the body to explain the *what*, *why*, *vs*, and *how*.
```

File: CONTRIBUTING.md
Comment (KukretiShubham) [inline CONTRIBUTING.md:56]: Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8405536)
```suggestion
* Do not end the subject line with a period.
```

File: CODE_OF_CONDUCT.md
Comment (KukretiShubham) [inline CODE_OF_CONDUCT.md:21]: Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8405536)
```suggestion
* Being respectful of differing opinions, viewpoints, and experiences
```

File: CONTRIBUTING.md
Comment (KukretiShubham) [inline CONTRIBUTING.md:52]: Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8405536)
```suggestion
* Use the imperative mood. (Example: "Move item to...", instead of "Moves item to...")
```

File: README.md
Comment (KukretiShubham) [inline README.md:5]: Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8405536)
```suggestion
* Uses modern web technologies
```

File: CONTRIBUTING.md
Comment (KukretiShubham) [inline CONTRIBUTING.md:66]: Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8405536)
```suggestion
* Fork the repository and create a branch for any issue that you are working on.
```

File: README.md
Comment (KukretiShubham) [inline README.md:9]: Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8405536)
```suggestion
* Handcrafted UI with UX in mind
```

File: CONTRIBUTING.md
Comment (KukretiShubham) [inline CONTRIBUTING.md:40]: Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8405536)
```suggestion
* Maintain the project's quality
```

File: README.md
Comment (KukretiShubham) [inline README.md:7]: Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8405536)
```suggestion
* Source compiles into static HTML and CSS with no/little JavaScript
```

File: CONTRIBUTING.md
Comment (KukretiShubham) [inline CONTRIBUTING.md:41]: Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8405536)
```suggestion
* Fix problems that are important to users
```

File: CODE_OF_CONDUCT.md
Comment (KukretiShubham) [inline CODE_OF_CONDUCT.md:20]: Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8405536)
```suggestion
* Demonstrating empathy and kindness toward other people
```

File: CONTRIBUTING.md
Comment (KukretiShubham) [inline CONTRIBUTING.md:94]: Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8405536)
```suggestion
* Feature Updates: `feat-<2-3-Words-Description>-<ISSUE_NO>`
```

File: README.md
Comment (KukretiShubham) [inline README.md:18]: Codacy has a fix for the issue: [Expected: 1; Actual: 2](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8405536)
```suggestion
1. Skip all of the rest steps and use Gitpod, which would automatically do all of this for you: [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/web3community/devprotocol.xyz)
```

File: CONTRIBUTING.md
Comment (Panquesito7) [inline CONTRIBUTING.md:43]: Wrong.

File: CONTRIBUTING.md
Comment (Panquesito7) [inline CONTRIBUTING.md:70]: Wrong.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Pretty much all the changes here are wrong 😅 they break the Markdown linter standards.

**Comment by krishguptadev** [comment]
This PR is all about switching from markdownlint to remark?

**Comment by Panquesito7** [comment]
I don't like these changes at all. Markdown Linter is better, IMO.
Let's see what the other maintainers think about this, though. 🙂

**Comment by krishguptadev** [comment]
> I don't like these changes at all. Markdown Linter is better, IMO. Let's see what the other maintainers think about this, though. 🙂

Remark is the recommended linter by **codacy**, I think the people at codacy know better about styling and stuff..

Plus, if we don't like these changes in the future we can revert them back any time 🙂

**Comment by krishguptadev** [comment]
Also, note: Codacy build will always fail since settings need to be changed after this pr is merged

**Review by vinzvinci — COMMENTED** [review]
Hmm, I'm not aware of remarklint. Maybe some maintainer can also review

**Comment by krishguptadev** [comment]
> Hmm, I'm not aware of remarklint. Maybe some maintainer can also review

Not sure why do we need to care so much about the formatter for markdown 🤔 I think remark is fine, else we can switch to markdown lint but it is a ruby package and just makes the contribution process harder 😕. For @Panquesito7's thought, it is not the linter it is the config, feel free to change the config 🙂

**Comment by KukretiShubham** [comment]
> > Hmm, I'm not aware of remarklint. Maybe some maintainer can also review
> 
> Not sure why do we need to care so much about the formatter for markdown 🤔 I think remark is fine, else we can switch to markdown lint but it is a ruby package and just makes the contribution process harder 😕. For @Panquesito7's thought, it is not the linter it is the config, feel free to change the config 🙂

I have no idea about format remarklint. Can we implement and see changes, incase we don't like the practice, we can revert back? 🤔

**Comment by shuklaritvik06** [comment]
> > > Hmm, I'm not aware of remarklint. Maybe some maintainer can also review
> > 
> > 
> > Not sure why do we need to care so much about the formatter for markdown thinking I think remark is fine, else we can switch to markdown lint but it is a ruby package and just makes the contribution process harder confused. For @Panquesito7's thought, it is not the linter it is the config, feel free to change the config slightly_smiling_face
> 
> I have no idea about format remarklint. Can we implement and see changes, incase we don't like the practice, we can revert back? thinking

Same +1

**Comment by Panquesito7** [comment]
For me, the Markdown linter is already good, I don't see a need to change. I think we're good as-is, it doesn't affect anything, IMO. 🙂

**Comment by krishguptadev** [comment]
> For me, the Markdown linter is already good, I don't see a need to change. I think we're good as-is, it doesn't affect anything, IMO. 🙂

At first, I was implementing markdown linter but it has some weird requirements :/ You can't make a line have more than 80 chars and more like there can't be inline HTML... That is why I switched to using **remark**. Moreover, remark is the preferred linter by codacy and most JS projects 🙂

**Review by Panquesito7 — APPROVED** [review]

**Comment by krishguptadev** [comment]
@vinzvinci awaiting your approval...

**Review by shuklaritvik06 — APPROVED** [review]
I am approving as we discussed this :)

**Review by vinzvinci — APPROVED** [review]

**Pull request merged**
